packages/ui-app/src/util/isValidBalanceExpectedResponses.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { IsValidWithMessage } from './types';
+
+import { TranslationFunction } from 'i18next';
+
+export enum ErrorMessage {
+  MustBeStringType = 'Balance input value must be valid type',
```

Comment:
As soon as a file is available that 

(a) is not a test
(b) has naked strings

We end up with future issues.